snd-server/genres_classificator.py

```diff
@@ -0,0 +1,39 @@
+import json
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+from context import Context
+
+
+class GenresClassificator:
+    def __init__(self, ctx: Context):
+        self.config = ctx.config
+        self.tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+        self.ids_to_labels = dict()
+
+    async def load_model(self):
+        self.model = AutoModelForSequenceClassification.from_pretrained(
+            "./bert_fine_tuned_genres"
+        )
+        with open("./bert_fine_tuned_genres/config.json", "r") as file:
+            self.ids_to_labels = json.load(file)["id2label"]
+
+    async def predict(self, prompt_text: str):
+        await self.load_model()
```

Comment:
Can this go into some `init` method, so we only do it once? We can call it at `app.py` in startup method for example